src/application/use-cases/tokenService/createToken/createToken.service.ts

```diff
@@ -0,0 +1,42 @@
+import jwt from 'jsonwebtoken';
+import { v4 as uuidv4 } from 'uuid';
+
+import { getJWT } from '../../../../config';
+import { JWT_ACCESS_TOKEN_LIFETIME, JWT_REFRESH_TOKEN_LIFETIME } from '~/back-constants/index';
+import { AdminTokenPayload, RefreshTokenPayload } from '~/back-shared/types/tokens/types';
+
+export const createTokenService = () => ({
+  generateTokens: (admin: { id: string; type: string }) => {
+    const accessTokenJti = uuidv4();
+    const refreshTokenJti = uuidv4();
+
+    const accessTokenPayload: Omit<AdminTokenPayload, 'refreshJti'> = {
+      id: admin.id,
+      type: admin.type,
+      jti: accessTokenJti
+    };
+
+    const accessToken = jwt.sign({ ...accessTokenPayload, refreshJti: refreshTokenJti }, getJWT.JWT_ACCESS_TOKEN, {
```

Comment:
BTW function `sign` has `jwtid` in Signoptions which allows inject jti into jwt token. 

<img width="518" height="366" alt="Image" src="https://github.com/user-attachments/assets/dc805e27-b651-4418-986b-3b701569b96b" />